examples/ColorPicker.advancedPopover.tsx

```diff
@@ -69,7 +69,10 @@ export default () => {
           visible={isOpen}
           placement='bottom-start'
         >
-          <IconButton onClick={() => setIsOpen((open) => !open)}>
+          <IconButton
```

Comment:
nit: On the below `<Button>` on L82, you can add an `aria-label` with the capitalized color string (similar to https://github.com/iTwin/iTwinUI/pull/1470#discussion_r1287280570)

Reply:
that button does not need an aria-label, it already has text content

Reply:
then the displayed value should be fixed, instead of slapping an aria-label on it.

further reading (if interested):
- https://ericwbailey.design/published/aria-label-is-a-code-smell/
- https://benmyers.dev/blog/overriding-screenreader-pronunciations/

Reply:
Thanks for the articles. Upon reading it, I am inclined towards leaving it as it is.

Because when users hear "his-lea", they can guess that it means "ache-es-el-ea". By adding an `aria-label` to clear this ambiguity, we might be making the braille display harder to read. And also it may lead to errors where we change the text content but forget to update the `aria-label`.